server/scsynth/SC_Jack.cpp

```diff
@@ -305,6 +305,9 @@ void SC_JackDriver::ConnectClientOutputs(const char* pattern) {
     free(ports);
 }
 
+
+static inline double sc_JACKOSCTimeSeconds() { return (uint64)OSCTime(getTime()) * kOSCtoSecs; }
+#define SC_JACK_USE_DLL
```

Comment:
can you please do what the preprocessor would do anyway and remove all the code sections that are now turned off by this? then remove this line.